drivers/sensor/ism330dhcx/ism330dhcx.c

```diff
@@ -750,13 +750,8 @@ static const struct ism330dhcx_config ism330dhcx_config = {
 #else
 #error "BUS MACRO NOT DEFINED IN DTS"
 #endif
-#ifdef CONFIG_ISM330DHCX_TRIGGER
-	/* One gpio pin declared in DTS */
-	.int_gpio_port = DT_INST_GPIO_LABEL(0, drdy_gpios),
-	.int_gpio_pin = DT_INST_GPIO_PIN(0, drdy_gpios),
-	.int_gpio_flags = DT_INST_GPIO_FLAGS(0, drdy_gpios),
-	.int_pin = DT_INST_PROP(0, int_pin),
```

Comment:
I see, `int_pin` is being deleted here, it seems it indicated whether to use INT1/2.

Reply:
indeed, we should probably use `drdy` and not `int` for the actual gpios, this driver is confusing.

Reply:
Fixed!